Kajabi/sage-lib
# Remove a breaking change for SagePanelControls sort

## Description

This PR is to add back the sort conditional in Sage Panel Controls.

## Testing in `sage-lib`
Nothing changes in `sage-lib`

## Testing in `kajabi-products`
1. Navigate to Podcasts / Episodes 
2. Confirm Sort in the panel controls still functions
3. Navigate to Website / Pages / Landing Pages
4. Confirm Sort in the panel controls still functions
